frontend/types/types.ts

```diff
@@ -373,3 +373,8 @@ export type ReviewRequestInterface = {
   createdAt: string
   updatedAt: string
 } & PartialReviewRequestInterface
+
+export interface UserSettings {
+  dn: string
+  theme: string
```

Comment:
It's hard to keep track of what `theme` means throughout this PR. Can we clarify that this is a `themeKey`? Sorry if this results in wide sweeping changes, but being able to tell that this is this is linked to the `key` will help with readability.